msteams-platform/sbs-graphactivity-feedbroadcast.yml

```diff
@@ -0,0 +1,381 @@
+### YamlMime:Tutorial
+title: Graph activity feed broadcast
+metadata:
+  title: Graph activity feed broadcast
+  description: In this tutorial, you'll learn to broadcast a message to all members of organization using activity feed notification.
+  audience: Developer
+  level: Beginner
+  ms.date: 04/08/2022
+  ms.topic: interactive-tutorial
+  nextTutorialHref: concepts/design/activity-feed-notifications.md
+  nextTutorialTitle: Read more on activity feed notifications.
+  ms.custom: mvc
+  ms.localizationpriority: high
+items:
+- durationInMinutes: 1
+  content: |
+    The Microsoft Teams activity feed enables users to triage items that require attention by notifying them of changes. You can use the activity feed notification APIs in Microsoft Graph to extend this functionality to your app.
+
+    This step-by-step guide shows a feature where users can broadcast a message to all members of the organization using activity feed notification.
+
+    **Interactions of graph activity feed broadcast**
+
+    * Broadcasts message for all the members.
+    * Displays activity feed notification.
+    * Broadcasts details available for members. 
+
+    You'll see the following output:
+
+      :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/outputready.png" alt-text="Output of feed broad cast" border="true":::
+
+- title: Prerequisites
+  durationInMinutes: 1
+  content: |
+    To set up your development environment, install the following:  
+
+    * [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+    * Latest version of Visual Studio
+    * [ngrok](https://ngrok.com/download) latest version (only for devbox testing) or any equivalent tunneling solution
+    
+      > [!NOTE]
+      > After downloading ngrok, sign up and install [authtoken](https://ngrok.com/download).
+    
+    * [Teams](https://teams.microsoft.com/) with valid account
+
+- title: Set up local environment
+  durationInMinutes: 1
+  content: |
+   Clone `Microsoft-Teams-Samples` repository to your local GitHub:  
+
+    1. Open [Microsoft Teams Samples](https://github.com/OfficeDev/Microsoft-Teams-Samples).
+    1. Select **Code**.
+    1. From the dropdown menu, select **Open with GitHub Desktop**.
+
+        :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/clonerepository.png" alt-text="Clone repository" border="true":::
+
+    1. Select **Clone**. 
+
+- title: Register your Teams auth single sign-on (SSO) with Azure Active Directory (Azure AD)
```

Comment:
Please do not use names and abbreviations in the header. You can do that in the content, not for heading.
Rework on the heading.

Reply:
Header changed.